DigitalLearningSolutions.Data/Services/ProgressService.cs

```diff
@@ -12,6 +12,8 @@ public interface IProgressService
         void UpdateCompleteByDate(int progressId, DateTime? completeByDate);
 
         void UpdateCompletionDate(int progressId, DateTime? completionDate);
+
+        public void UpdateDiagnosticScore(int progressId, int tutorialId, int myScore);

```

Comment:
`public` access modifier not required here.

Reply:
removed